### PR TITLE
Vtcca 0 play 2.6

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

File: app/uk/gov/hmrc/voapropertylinking/controllers/ReverseRoutes.scala [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+import play.api.mvc.Call
+
+
+import _root_.uk.gov.hmrc.voapropertylinking.binders.propertylinks._
+import _root_.uk.gov.hmrc.voapropertylinking.binders.propertylinks.temp._
+import _root_.scala.language.reflectiveCalls
+import _root_.models._
+
+// @LINE:3
+package uk.gov.hmrc.voapropertylinking.controllers
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/voapropertylinking/controllers/ReverseRoutes.scala]: I've never seen ReverseRoutes appear in a PR before.

File: app/uk/gov/hmrc/voapropertylinking/controllers/ReverseRoutes.scala [hunk 2]
```diff
@@ -0,0 +1,314 @@
+// @GENERATOR:play-routes-compiler
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/controllers/ReverseRoutes.scala]: whoa. this generated pile of code shouldn't have been added to git. can you check how this happened?

File: app/uk/gov/hmrc/voapropertylinking/controllers/javascript/JavaScriptReverseRoutes.scala [hunk 3]
```diff
@@ -0,0 +1,464 @@
+// @GENERATOR:play-routes-compiler
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/controllers/javascript/JavaScriptReverseRoutes.scala]: same here. not sure how this ended up in git

File: app/uk/gov/hmrc/voapropertylinking/controllers/routes.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+// @GENERATOR:play-routes-compiler
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/controllers/routes.java]: and here

File: app/uk/gov/hmrc/voapropertylinking/controllers/test/ReverseRoutes.scala [hunk 5]
```diff
@@ -0,0 +1,41 @@
+// @GENERATOR:play-routes-compiler
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/controllers/test/ReverseRoutes.scala]: same here... did you somehow break build.sbt? looks like play plugin is misconfigured and outputs generated router code into our code base

File: app/uk/gov/hmrc/voapropertylinking/controllers/test/javascript/JavaScriptReverseRoutes.scala [hunk 6]
```diff
@@ -0,0 +1,55 @@
+// @GENERATOR:play-routes-compiler
+// @SOURCE:/Users/danielconnelly/Projects/HMRC/VOA/voa-property-linking/conf/test.routes
+// @DATE:Wed Jan 08 12:36:25 GMT 2020
+
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/controllers/test/javascript/JavaScriptReverseRoutes.scala]: one more

File: project/MicroServiceBuild.scala [hunk 7]
```diff
@@ -15,15 +15,28 @@ private object AppDependencies {
...
-    "uk.gov.hmrc" %% "play-reactivemongo" % "5.2.0",
     ws,
-    "ai.x" %% "play-json-extensions" % "0.9.0",
-    "uk.gov.hmrc" %% "auth-client" % "2.5.0",
-    "uk.gov.hmrc" %% "bootstrap-play-25" % "4.13.0",
-    "uk.gov.hmrc" %% "mongo-lock" % "5.1.1",
-    "uk.gov.hmrc" %% "domain" % "4.1.0",
-    "org.typelevel" %% "cats-core" % "1.6.1",
-    "uk.gov.hmrc" %% "reactive-circuit-breaker" % "2.1.0"
+    guice,
```
Comment (PeterPerhac) [inline project/MicroServiceBuild.scala]: guice is here twice, looks like possibly more dependencies got duplicated.

**Comment by reddy-yattapu** [comment]
:+1:

**Comment by PeterPerhac** [comment]
👍 LOOKS GOOD TO ME

**Comment by darrell-plant** [comment]
lgtm 👍

**Pull request merged**